.travis.yml

```diff
@@ -4,9 +4,6 @@ language: cpp
 os:
   - linux
   - osx
-compiler:
-  - clang
```

Comment:
Ah, you mean that `TEST_GROUP=1` and `TEST_GROUP=2` don't run on OSX? I can enable those.